README.md

```diff
@@ -6,7 +6,7 @@ High-performance spatio-temporal data analysis platform for OpenStreetMap full-h
 <img src="https://heigit.org/wp-content/uploads/2018/01/HeiGIT_Logo_cut-505x100@2x.png" height="80px" width="404px">
 
 [![Build Status](https://jenkins.ohsome.org/buildStatus/icon?job=oshdb/master)](https://jenkins.ohsome.org/blue/organizations/jenkins/oshdb/activity/?branch=master)
-[![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.heigit.bigspatialdata/oshdb/badge.svg)](https://search.maven.org/artifact/org.heigit.bigspatialdata/oshdb)
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.heigit.ohsome/oshdb/badge.svg)](https://search.maven.org/artifact/org.heigit.ohsome/oshdb)
```

Comment:
I know, strictly speaking this is not related to this PR, but I forgot to update this in the `0.6.0` release. And IMHO it's better to merge this sooner than later :wink: